ocaml/xcp-rrdd/bin/rrdp-iostat/rrdp_iostat.ml

```diff
@@ -998,7 +1017,7 @@ let gen_metrics () =
               None
           | Some s ->
               if Hashtbl.mem s sr_vdi then
-                Some (Hashtbl.find s sr_vdi)
+                Hashtbl.find s sr_vdi
```

Comment:
The combination of mem/find is prone to race conditions and not efficient. It's better to use `Hashtbl.find_opt`; I am surprised that the current code is type correct.

Reply:
True, I'll change it to `find_opt`.